packages/react-component-library/src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -345,24 +345,26 @@ describe('DatePicker', () => {
       )
     })
 
-    describe('when Shift-Tab is pressed twice', () => {
+    describe('when Shift-Tab is pressed three times', () => {
       beforeEach(async () => {
         await user.tab({ shift: true })
         await user.tab({ shift: true })
+        await user.tab({ shift: true })
       })
 
       it('traps the focus within the picker', () => {
         expect(wrapper.getByText('1')).toHaveFocus()
       })
 
       describe('and Tab is then pressed once', () => {
-        beforeEach(() => {
-          return user.tab()
+        beforeEach(async () => {
+          await user.tab({ shift: true })
         })
 
         it('still traps the focus within the picker', () => {
-          const dayPicker =
-            wrapper.container.querySelectorAll('.DayPicker-wrapper')[0]
+          const dayPicker = wrapper.container.querySelectorAll(
+            '.DayPicker-NavButton--next'
+          )[0]
```

Comment:
Hmm yeah ok this ones annoying.

We might need to do something about the tabindex so that the 'Jump to date' button is focussed initially.

Once the 1st day is focussed you can use the arrow keys to navigate between the different days (intended).

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex